cmd/otelcol/config/collector/fargate_config.yaml

```diff
@@ -0,0 +1,153 @@
+    # TODO: Find and add receivers equivalent to missing monitors collectd/protocols and vmem.
+    scrapers:
+      cpu:
+      disk:
+      filesystem:
+      memory:
+      network:
+      # System load average metrics https://en.wikipedia.org/wiki/Load_(computing)
+      load:
+      # Paging/Swap space utilization and I/O metrics
+      paging:
+      # Aggregated system process count metrics
+      processes:
+      # System processes metrics, disabled by default
+      # process:
+  smartagent/signalfx-forwarder:
+    type: signalfx-forwarder
+    listenAddress: 0.0.0.0:9080
+  smartagent/ecs-metadata:
+    type: ecs-metadata
+    excludedImages: ${env:IMAGES_TO_EXCLUDE}
+
+processors:
+  batch:
+  # Enabling the memory_limiter is strongly recommended for every pipeline.
+  # Configuration is based on the amount of memory allocated to the collector.
+  # In general, the ballast should be set to 1/3 of the collector's memory, the limit
+  # should be 90% of the collector's memory up to 2GB. The simplest way to specify the
+  # ballast size is set the value of SPLUNK_BALLAST_SIZE_MIB env variable. Alternatively, the
+  # --mem-ballast-size-mib command line flag can be passed and take priority.
+  # For more information about memory limiter, see
+  # https://github.com/open-telemetry/opentelemetry-collector/blob/main/processor/memorylimiter/README.md
+  memory_limiter:
+    ballast_size_mib: ${SPLUNK_BALLAST_SIZE_MIB}
+    check_interval: 2s
+    limit_mib: ${SPLUNK_MEMORY_LIMIT_MIB}
+  # detect if the collector is running on a cloud system
+  # important for creating unique cloud provider dimensions
+  resourcedetection:
+    detectors: [system, ecs]
+    override: false
+  # Same as above but overrides resource attributes set by receivers
+  resourcedetection/internal:
+    detectors: [system, ecs]
+    override: true
+  # Enables the filter processor with example settings
+  # Full configuration here: https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/filterprocessor
+  # NOTE: These settings need to be change when using this processor
+  filter/1:
+    metrics:
+      exclude:
+        match_type: regexp
+        metric_names: ${env:METRICS_TO_EXCLUDE}
+
+exporters:
+  # Traces
+  sapm:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    endpoint: "https://ingest.${SPLUNK_REALM}.signalfx.com/v2/trace"
+  # Metrics + Events
+  signalfx:
+    access_token: "${SPLUNK_ACCESS_TOKEN}"
+    realm: "${SPLUNK_REALM}"
+    sync_host_metadata: true
+    correlation:
+#  # Logs
+#  splunk_hec:
+#    token: "${SPLUNK_HEC_TOKEN}"
+#    endpoint: "${SPLUNK_HEC_URL}"
+#    source: "otel"
+#    sourcetype: "otel"
+  # Debug
+  logging:
+    loglevel: ${env:LOG_LEVEL}
+
+service:
+  extensions: [health_check, http_forwarder, zpages]
+  pipelines:
+    traces:
+      receivers: [jaeger, sapm, zipkin, smartagent/signalfx-forwarder]
+      processors: [memory_limiter, batch, resourcedetection]
+      #- resource/add_environment
```

Comment:
If we have this processor usage commented out, let's add its configuration in the `processors` section above, commented out as well.

Reply:
Ok